tests/integration/hardhatprocess.nim

```diff
@@ -63,7 +63,28 @@ method start*(node: HardhatProcess) {.async.} =
     node.process = await startProcess(
       node.executable,
       node.workingDir,
-      @["node", "--export", "deployment-localhost.json"].concat(node.arguments),
+      @["node"].concat(node.arguments),
+      options = poptions,
+      stdoutHandle = AsyncProcess.Pipe,
+    )
+
+    discard await startProcess(
+      node.executable,
+      node.workingDir,
+      @["node", "run", "scripts/mine.js", "--network", "localhost"].concat(
+        node.arguments
+      ),
+      options = poptions,
+      stdoutHandle = AsyncProcess.Pipe,
+    )
+
+    discard await startProcess(
+      node.executable,
+      node.workingDir,
+      @[
+        "node", "ignition", "deploy", "ignition/modules/marketplace.js", "--network",
+        "localhost",
+      ].concat(node.arguments),
```

Comment:
Instead of starting a (long running) process, it would be better to execute a command using `execCommandEx`, eg
```nim
let execResult = await execCommandEx("node run scripts/mine.js --network localhost")
```